src/newSeed/stage4.ts

```diff
@@ -34,6 +34,14 @@ export class Stage4 extends BaseStage {
     });
   }
 
+  toggleCheckbox(name: string): void {
+    if (name === "whitelist") {
+      this.seedConfig.seedDetails.whitelist.isWhitelist = !this.seedConfig.seedDetails.whitelist.isWhitelist;
+    } else {
+      this.seedConfig.seedDetails[name] = !this.seedConfig.seedDetails.geoBlock;
```

Comment:
ass this line is written, `[name]` will not work for multiple cases.  It is misleadingly generalized.  I would just explicitly use `.geoBlock` instead of `[name]`